src/Common/src/Interop/Windows/mincore/Interop.ConsoleScreenBufferInfo.cs

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class mincore
+    {
+        [StructLayoutAttribute(LayoutKind.Sequential)]
+        internal struct CONSOLE_SCREEN_BUFFER_INFO
```

Comment:
And ConsolePal.Windows.cs doesn't use GetConsoleScreenBufferInfo?  I'm just wondering in what situation GetConsoleScreenBufferInfo and CONSOLE_SCREEN_BUFFER_INFO would ever be used independently.  But feel free to go ahead and check-in like this if you think it's right.